configure.ac

```diff
@@ -32,6 +32,7 @@ AC_CONFIG_FILES([
   subsystems/mvtx/Makefile
   subsystems/tpc/Makefile
   subsystems/vertex/Makefile
+  subsystems/siliconseeds/Makefile
```

Comment:
Could we just make this `trackseeds` instead of `siliconseeds`? We should make a tpc seed and full svtx seed module too, and they could all live in here

Reply:
Done, change committed